packages/amplify-prompts/src/progressbars/progressbar.ts

```diff
@@ -0,0 +1,252 @@
+import { AmplifyTerminal as Terminal } from './terminal';
+
+/**
+ * Type for Bar configurations options
+ */
+export type BarOptions = {
+    // Custom formatter functions for main progress bar and item's underneath it
+    progressBarFormatter: (payload: ProgressPayload) => string,
+    itemFormatter: (payload: ItemPayload) => string,
+    // Indicates if the progress bar is part of a multiBar or standalone
+    loneWolf: boolean,
```

Comment:
This boolean gives me pause. Maybe we should always use `multibar` and a single progress bar is just a multibar of 1?